env_vars_32.sh

```diff
@@ -0,0 +1,4 @@
+# Environment variables for build
+OPENBLAS_VERSION="v0.3.7"
+MACOSX_DEPLOYMENT_TARGET=10.9
+CFLAGS="-msse2 -std=c99 -fno-strict-aliasing"
```

Comment:
This is comes from `env_vars.sh`, correct? I assume the extra bit is `-msse2`. I assume the latest multibuild is updated to use this. Is it documented in the README?

Reply:
The needed bit is the `-msse2`. Commenting here, do you have a place in the README you would like to see it documented as well?

Reply:
Added a line to the README, as well as explaining the OpenBLAS download